Doc/faq/windows.rst

```diff
@@ -276,3 +276,9 @@ How do I check for a keypress without blocking?
 Use the :mod:`msvcrt` module.  This is a standard Windows-specific extension module.
 It defines a function ``kbhit()`` which checks whether a keyboard hit is
 present, and ``getch()`` which gets one character without echoing it.
+
+How do I solve the missing api-ms-win-crt-runtime-l1-1-0.dll error?
+-------------------------------------------------------------------
+
+This can occur on Python 3.5 and later, and on Windows 8.1 or earlier.
+Install the `Update for Universal C Runtime <https://support.microsoft.com/en-us/help/3118401/>`_.
```

Comment:
```suggestion
This can occur on Python 3.5 and later, and on Windows 8.1 or earlier when not all updates have been installed.
First ensure your operating system is supported, is up to date, and if that does not resolve the issue,
visit the `Microsoft support page <https://support.microsoft.com/en-us/help/3118401/>`_
for guidance on manually installing the C Runtime update.
```